Drop-In/src/main/java/com/braintreepayments/api/DropInActivity.java

```diff
@@ -209,7 +209,7 @@ public void onResult(@Nullable PaymentMethodNonce deletedNonce, @Nullable Except
                     //mLoadingViewSwitcher.setDisplayedChild(0);
                 } else {
                     getDropInClient().sendAnalyticsEvent("manager.unknown.failed");
-                    // TODO: determine how to handle unexpected error when deleting payment method (previously finished drop in)
+                    onError(error);
```

Comment:
Decided to go with the same flow as the previous versions, seems good to give merchants information on the error and they can handle it how they choose.